Mac-App/Backend/BackendTests/Parsers/SwiftDeps/SwiftDepsParserSpec.swift

```diff
@@ -14,15 +14,39 @@ final class SwiftDepsParserSpec: QuickSpec {
     override func spec() {
         describe("parseSwiftDeps") {
             context("given it takes a valid input") {
-                it("returns correctly parsed DependencyTree objects") {
-                    let result = parseSwiftDeps(
-                        parseYamlUrls(
-                            from: findProjectOutputDirectories(projectName: "Backend")
+                var result: [DependencyTree] = []
+
+                beforeEach {
+                    // Do not perform same search for each test case as the results are the same
+                    // And it is expensive & long
+                    if result.isEmpty {
```

Comment:
This is a bit ugly but did not think about it too much as it's a test :) 
But any suggestions welcome!